### ISSUE TITLE
Using Mnemonic Phrase to import the wallet

### ISSUE THREAD

**Issue by heawen110** [AfterworkBlockchain/GenesisMobile#18]
**TL;DR**

We will not support import private key from mnemonic phrase until the Expo SDK team update to the new infrastructure, which is expected to be delivered in February.

-----

**Detail:**

To get private key from mnemonic phrases we need generate seed with [react-native-randomBytes](https://github.com/mvayngrib/react-native-randombytes) modules, which is explained on [stackoverflow](https://ethereum.stackexchange.com/questions/39384/how-to-generate-private-key-public-key-and-address)

```
const bip39 = require('bip39');
const hdkey = require('ethereumjs-wallet/hdkey');

const mnemonic = '..';
const hdwallet = hdkey.fromMasterSeed(bip39.mnemonicToSeed(mnemonic));
const path = "m/44'/60'/0'/0/0";
const wallet = hdwallet.derivePath(path).getWallet();
const address = `0x${wallet.getAddress().toString('hex')}`;

console.log(`Address: ${address}`);
```

to generate randomBytes we need to [access to Native Module of React Native](https://github.com/expo/expo/issues/56). Currently, need to write native code to build a feature that can’t be implemented using the current set of primitives provided by React Native & the Expo SDK. We eject the project from ExpoKit, and it will introduce a lot of complexity and slow down the developing.

The way to do it is stated [here](https://docs.expo.io/versions/v28.0.0/expokit/detach/#__next). 

As the Expo team already realizes this problem and[ is developing on that](https://blog.expo.io/expokit-2019-1e5cb02106f8),

> we will add an option to eject to a plain React Native app with most Expo APIs installed in the project but none of the infrastructure for versioning, multiplexing, or code & asset delivery. 

The expecting delivery time is in February, so we will keep waiting for that, and we believe the current private key import function could be enough for most of the use cases.

We will currently not implement this function, but rather waiting for the next update of Expo SDK custom mode.
